browse/domain/metadata.py

```diff
@@ -256,3 +218,23 @@ def get_datetime_of_version(
             return None
         else:
             return versions[0].submitted_date
+        
+    def display_secondaries(self)-> List[str]:
+        """Unalias, dedup and sort secondaries for display."""
+        if not self.secondary_categories:
+            return []
+
+        def unalias(secs): # type: ignore
```

Comment:
Just out of curiosity, why the # type: ignores in this area? Maybe add a comment about why if possible.